connector/subgraph/schema.graphql

```diff
@@ -107,10 +115,22 @@ type Token @entity {
   decimals: Int!
 }
 
+type CollateralRequirement @entity {
+  id: ID!
+  proposal: Proposal
+  token: Token
+  actionAmount: BigInt
+  challengeAmount: BigInt
+  challengeDuration: BigInt
+}
+
 enum ProposalStatus {
   Active
   Cancelled
+  Challenged
+  Disputed
   Executed
+  Rejected
```

Comment:
Let's add the new statuses below the existing ones, so we keep the same status num for the existing values.